app/views/unattended/poap/POAP.erb

```diff
@@ -0,0 +1,9 @@
+<%#
```

Comment:
I am assuming Cisco scritps are proprietary since they are only distributed in their closed support site. I could not download them with a simple "guest" account not associated with a support contract.
I will write a new sample script in a couple of days, since I think that having a default template would simplify the use of this feature.